rust/main/utils/run-locally/src/invariants/termination_invariants.rs

```diff
@@ -126,7 +126,7 @@ pub fn termination_invariants_met(
         "Verbose logs not expected at the log level set in e2e"
     );
 
-    let gas_payment_sealevel_events_count = fetch_metric(
+    let _gas_payment_sealevel_events_count = fetch_metric(
```

Comment:
you should be able to remove this altogether now, since it's not used

Reply:
Fixed